src/test/javascript/spec/component/drag-and-drop-question/drag-and-drop-question-edit.component.spec.ts

```diff
@@ -66,7 +78,7 @@ describe('DragAndDropQuestionEditComponent', () => {
         tick();
     }));
 
-    afterEach(function () {
```

Comment:
the `fakeAsync` in the beforeEach, as well as `tick()` and `fixture.detectChanges(false)` are required I guess? What does `fixture.detectChanges(false)` mean anyways? Do you know?

Reply:
Yes, they are required (at least the tests fail without them).
`fixture.detectChanges(false)` seems to detect any changes but not fail if there were changes.

`detectChanges(checkNoChanges?: boolean): void;`
true seems to be the default